Packs/Core/Playbooks/playbook-Office_process_creates_a_scheduled_task_via_file_access.yml

```diff
@@ -0,0 +1,1638 @@
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "105":
+    id: "105"
+    taskid: 9e626d73-5635-4c35-8467-c98288a627b8
+    type: regular
+    task:
+      id: 9e626d73-5635-4c35-8467-c98288a627b8
+      version: -1
+      name: Execute shell command to get the office file hash
+      description: Calculate the Office file Hash by executing shell commands.
+      script: '|||core-run-script-execute-commands'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "106"
+    scriptarguments:
+      commands:
+        simple: certutil -hashfile "${OfficeFilePath}" sha256
+      endpoint_ids:
+        simple: ${alert.agentid}
+      extend-context:
+        simple: GetHashCommand=
+      ignore-outputs:
+        simple: "true"
+      timeout:
+        simple: "120"
+      timeout_in_seconds:
+        simple: "120"
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: errorPath
+    view: |-
+      {
+        "position": {
+          "x": -210,
+          "y": -675
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "106":
+    id: "106"
+    taskid: fb1965c8-a114-449b-8bd5-1b117f06ef68
+    type: regular
+    task:
+      id: fb1965c8-a114-449b-8bd5-1b117f06ef68
+      version: -1
+      name: Get script execution results
+      description: Retrieve the results of a script execution action.
+      script: '|||core-get-script-execution-results'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "107"
+    scriptarguments:
+      action_id:
+        complex:
+          root: GetHashCommand
+          accessor: action_id
+          transformers:
+          - operator: uniq
+    reputationcalc: 2
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -210,
+          "y": -510
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "107":
+    id: "107"
+    taskid: d56e6c57-7020-4d2f-834e-0b7c0372569e
+    type: regular
+    task:
+      id: d56e6c57-7020-4d2f-834e-0b7c0372569e
+      version: -1
+      name: Set Office file hash
+      description: |-
+        Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.10/Cortex-XSOAR-Administrator-Guide/Automations
```

Comment:
Is this only for XSOAR 6?  What about XSOAR 8 Cloud, XSOAR OPP and XSIAM.
For Cortex XSOAR 6.13, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.13/Cortex-XSOAR-Administrator-Guide/Licenses
 For Cortex XSOAR 8 Cloud, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Cloud-Documentation/Scripts
For Cortex XSOAR 8.7 OPP, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8.7/Cortex-XSOAR-On-prem-Documentation/Get-Started-with-Cortex-XSOAR
For Cortex XSIAM, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Documentation/Automations